### PR TITLE
add pre build and post build actions in order for travis to execute t…

### PR DESCRIPTION
**Pull request by germanStabile**
…est reporter actions

### CODE REVIEW

**Comment by toptierlabs** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">PR is classed as Work in Progress. Do NOT merge</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .travis.yml [hunk 1]
```diff
@@ -24,4 +24,4 @@ script:
 after_script:
   - slather coverage -s --scheme ios-base ios-base.xcodeproj
   - bash <(curl -s https://codecov.io/bash) -f cobertura.xml -X coveragepy -X gcov -X xcode
-  - ./cc-test-reporter after-build -t cobertura --exit-code $TRAVIS_TEST_RESULT
+  - ./cc-test-reporter after-build --debug -t cobertura --exit-code $TRAVIS_TEST_RESULT
```
Comment (germanStabile) [inline .travis.yml]: 🙏

Comment (germanStabile) [reply .travis.yml]: verbose was useless, lets hope debug is better 😂

Comment (glm4) [reply .travis.yml]: lets see..

**Pull request merged**